patterns/2-structured/maturity-model.md

```diff
@@ -5,15 +5,15 @@ Maturity Model
 ## Patlet
 
 Teams have started adopting InnerSource. The practice is spreading to multiple
-departments. Understanding of what constitutes an InnerSource project are wide
-spread though. The solution is to provide a maturity model to allow for teams to
+departments. However, the understanding of what constitutes an InnerSource project
+is not widespread. The solution is to provide a maturity model to allow for teams to
```

Comment:
```suggestion
departments. However, the understanding of what constitutes an InnerSource project
varies. The solution is to provide a maturity model to allow for teams to
```

Based on the [feedback](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/315/files#r661315766) from @MaineC I a modifying this sentence slightly.

The important piece here seems to be that different teams have different understandings of what InnerSource means. I also confirmed that this is the intended mean in the Problem statement, where it states: 
> ... the depth of understanding for the [InnerSource] concept will diverge ...

Given that, I think the change proposed here is easier to understand.